### ISSUE TITLE
ERROR: UndefVarError: add_constraint not defined

### ISSUE THREAD

**Issue by weiqiao** [lanl-ansi/MomentOpt.jl#20]
When I tried to add constraints like 
`@constraint gmp Mom(1, mu) == 1`
as in https://juliahub.com/docs/MomentOpt/Nuha4/0.2.0/started/
I always got error 
```
WARNING: both SumOfSquares and JuMP export "add_constraint"; uses of it in module MomentOpt must be qualified
ERROR: UndefVarError: add_constraint not defined
```
I have tried both Julia 1.5.3 and Julia 1.0.5 on both MacOS and Ubuntu 20.04.

**Comment by weiqiao** [comment]
@tweisser

**Comment by huihanl** [comment]
I have the same error as above: 

![image](https://user-images.githubusercontent.com/29032615/106406298-b7fad480-63ed-11eb-929b-ed9297820a9d.png)

I'm using Julia 1.5.3 on Mac. Thanks for looking that up! @tweisser

**Comment by blegat** [comment]
That was caused by https://github.com/jump-dev/SumOfSquares.jl/pull/174, should be resolved with https://github.com/jump-dev/SumOfSquares.jl/pull/183

**Comment by blegat** [comment]
This should be resolved with SumOfSquares v0.4.5, please reopen the issue if that's not the case

**Comment by weiqiao** [comment]
> This should be resolved with SumOfSquares v0.4.5, please reopen the issue if that's not the case

This solved the problem, thanks! Though I'm getting a different error.
https://github.com/lanl-ansi/MomentOpt.jl/issues/21
